### PR TITLE
fix: problem spacing

### PR DESCRIPTION
**Pull request by berkalpyakici**

### CODE REVIEW

File: generic-styles/index.less [hunk 1]
```diff
@@ -28,6 +28,15 @@
       margin-inline-start: 0;
     }
   }
+  code {
+    color: #c7254e;
+    background-color: #f9f2f4;
+    border-radius: 4px;
+  }
+  [data-type="composite-page"] [data-type="exercise"] [data-type="problem"] p:first-of-type a {
```
Comment (TomWoodward) [inline generic-styles/index.less]: is this the exact same selector that cnx uses? a little worried we're gonna miss some random type

File: generic-styles/index.less [hunk 2]
```diff
@@ -28,6 +28,15 @@
       margin-inline-start: 0;
     }
   }
+  code {
```
Comment (TomWoodward) [inline generic-styles/index.less]: this fix is on a different pr, should remove this commit from this branch

**Review by TomWoodward — CHANGES_REQUESTED** [review]

File: generic-styles/index.less (hunk 1 above)
Comment (berkalpyakici) [reply generic-styles/index.less]: CNX has [data-is-baked="true"] .media-body>#content [data-type="composite-page"] [data-type="exercise"] [data-type="problem"] p:first-of-type a {...}

Comment (TomWoodward) [reply generic-styles/index.less]: ok yea this is 👍 then

**Pull request merged**